README.md

```diff
@@ -76,6 +76,33 @@ values by `__SENSITIVE_DATA__` string. This feature is enabled by default but
 you can skip this (not recommanded) by setting the environment variable
 `LOGGER_USE_SENSITIVE_DATA_STREAM` to `false`.
 
+You also can add custom filter :
+```
+const sensitiveDataFragment = '(pass|password)'; // Will obfuscate 'pass' and 'password' data
+
+const newLogger = init({
+  logger: {
+    sensitiveDataFragment,
+  },
+});
+```
+
+Moreover, you can also add customize the way it replaces data :
+```
+const sensitiveDataPattern = [
+  {
+    regex: YOUR_NEW_REGEX,
+    substitute: SUBSTITUTION_CONTENT,
```

Comment:
I'm not sure we need to change the substitute I think we can forced to be __SENSITIVE_DATA__

Reply:
Indeed, it isn't realy about SENISITIVE_DATA but more about the whole pattern replacement such as :

`"password":"WHATEVER"` -> `"$1":"WHATEVERE"` which needs dots and quotes. 

Althought `password=whatever` - > `$1=WHATEVER` doesn't need.